src/lib/y2network/sysconfig/interfaces_reader.rb

```diff
@@ -87,14 +89,15 @@ def find_physical_interfaces
 
       # Finds the connections configurations
       def find_connections
-        @connections ||=
+        @connections =
```

Comment:
This is not an equivalent change, is it?

Reply:
no, it is not, but I analyze code and there is no change as result of this call is cached, so it was in fact double caching.